backend/routers/beers.py

```diff
@@ -1,81 +1,102 @@
-from fastapi import APIRouter, HTTPException, Request, Response
-from peewee import IntegrityError
-from backend.database import db
-from backend.models import Beer
+import os
 
-# from urllib.error import HTTPError
+from fastapi import APIRouter, Depends, HTTPException, Response, Request
+
+from sqlalchemy.orm import Session
+from sqlalchemy.exc import IntegrityError
+
+from backend.database import get_db
+from backend import models, schemas, settings
 
 from backend.utils.query_to_filters import query_to_filters
+from backend.utils.error_handler import response_from_error
+
+
+PAGE_LIMIT = int(os.getenv("BEER_PAGE_LIMIT", settings.BEER_PAGE_LIMIT))

```

Comment:
so you have a `settings` file and a `json` config file for the db params?

better have one file for all the settings and configs